### PR TITLE
Add LTW build task and Downloading LTW package task to the test pipelines.

### PR DESCRIPTION
**Pull request by dinghong2233**
Tested with LTW build pipeline. The auth-client-android-dev pipeline can trigger the LTW build pipeline correctly and the Android Broker CI](https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1490&_a=summary)
image

### CODE REVIEW

File: azure-pipelines/continuous-delivery/auth-client-android-dev.yml [hunk 1]
```diff
@@ -321,6 +321,24 @@ stages:
             filePath: '$(Build.SourcesDirectory)/azure-pipelines/scripts/queue-build.ps1'
             arguments: '-OrganizationUrl "https://msazure.visualstudio.com/" -Project "Intune" -PipelinePAT "$(pipelinepat)" -WaitTimeoutInMinutes 120 -BuildDefinitionId "237827" -PipelineVariablesJson "{ ''AdAccountsVersion'': ''$(brokerVersionNumber)'', ''CommonVersion'': ''$(versionNumber)'', ''MsalVersion'': ''$(versionNumber)'', ''AdalVersion'': ''$(versionNumber)'' }" -Branch "$(companyPortalBranch)"'
             workingDirectory: '$(Build.SourcesDirectory)'
+# BrokerApk - Link to Windows Queue pipeline
+- stage: 'queueLinkToWindowsPipeline'
```
Comment (iamgusain) [inline azure-pipelines/continuous-delivery/auth-client-android-dev.yml:325]: Can you add it under a condition, default to 'false'.
Once we enable e2e integration, we can remove the condition.

File: azure-pipelines/ui-automation/broker-test.yml [hunk 2]
```diff
@@ -46,7 +46,9 @@ variables:
   edgeApk: Edge.apk
   firebaseTimeout: 45m
   resultsHistoryName: Broker Release
-
+  OSServiceConnection: LTW-Integration
```
Comment (iamgusain) [inline azure-pipelines/ui-automation/broker-test.yml:49]: Who owns this service connection?

Comment (iamgusain) [reply azure-pipelines/ui-automation/broker-test.yml:49]: can you add permissions for "androidauth" SG

File: azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml [hunk 3]
```diff
@@ -24,7 +24,14 @@ parameters:
...
     default: 'AndroidAdal'
-
+  - name: LTWFeedName
+    type: string
+  - name: OSServiceConnection
+    displayName: Link to Windows Service Connection
+    type: string
+  - name: LTWVersion
+    displayName: Link to Windows Version
+    type: string
```
Comment (iamgusain) [inline azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml:34]: Let's add default values.

File: azure-pipelines/ui-automation/broker-test.yml (hunk 2 above)
Comment (dinghong2233) [reply azure-pipelines/ui-automation/broker-test.yml:49]: I created this and added the androidauth as admin of the connection.

File: azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml (hunk 3 above)
Comment (dinghong2233) [reply azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml:34]: Added for the LTWVersion

Comment (dinghong2233) [reply azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml:34]: Should we add for another 2 new-added vars? I see the old connection var does not have default value as well.

File: azure-pipelines/continuous-delivery/auth-client-android-dev.yml (hunk 1 above)
Comment (dinghong2233) [reply azure-pipelines/continuous-delivery/auth-client-android-dev.yml:325]: Added a variable to disable the LTW build by default: 
image

File: azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml (hunk 3 above)
Comment (dinghong2233) [reply azure-pipelines/ui-automation/templates/download-brokers-and-azure-sample.yml:34]: @iamgusain Please help review again. Thanks so much.

**Review by iamgusain — APPROVED** [review]
LGTM

**Review by somalaya — APPROVED** [review]

**Comment by iamgusain** [comment]
Can you please update the title of the PR before merging

**Comment by somalaya** [comment]
could you also add the changes to push the LTW apk to firebase device?
as shown in this snapshot? 
image

**Comment by dinghong2233** [comment]
image
Per Sowmya's suggestion, added the LTW app to the other file list of firebase task. Tested the apk was uploaded successfully. @iamgusain, @somalaya, please help review and let me know if any concern.

**Pull request merged**